.github/ISSUE_TEMPLATE/3-AZ-QUESTION.yml

```diff
@@ -0,0 +1,72 @@
+name: 🤔 Az module question or discussion
```

Comment:
If we want to promote https://github.com/Azure/azure-powershell/discussions , we can change this "az module question or discussion" template to a "contact_links"

Reply:
Bot doesn't support notify service team when labeling service attention on discussion item. Another limitation is author(reporter) will be changed when I transfer discussion to issue. So, for the time being, I'm hesitating to promote discussion.